src/print_event_cb.c

```diff
@@ -621,13 +627,17 @@ print_event_bin(void *cb_cookie, void *data, int size)
 {
 	(void) cb_cookie;
 
+	pthread_spin_lock(&OutputLock);
```

Comment:
On the other hand, if want to keep using our own lock, then let's not use libc's lock, so these `fwrite` calls should at least be changed to `fwrite_unlocked`.